UnitTestBot/UTBotJava
# Add missed mockInfoGenerator for system static objects

## Add missed mock information for static values of system classes

## Description

Removed a way to create a static object without explicitly providing `mockInfoGenerator`, added a required one for creating of system static objects

Fixes #2015

## How to test

### Automated tests

`org.utbot.examples.mock.MockStaticMethodExampleTest#testMockStaticMethodFromAlwaysMockClass`

### Manual tests

Add a system class to the `mockAlways` list, and run the generation of a code containing static method invocation. At least one branch should be found.

## Self-check list
